srcpkgs/gcc/template

```diff
@@ -366,6 +366,12 @@ do_install() {
 		rm -f ${DESTDIR}/usr/lib64
 	fi
 
+	# Remove "fixed" header
+	# https://gcc.gnu.org/bugzilla/show_bug.cgi?id=91085
+	if [ -e "${DESTDIR}/usr/lib/gcc/${_triplet}/${_minorver}/include-fixed/bits/statx.h" ]; then
+		rm -vrf ${DESTDIR}/usr/lib/gcc/${_triplet}/${_minorver}/include-fixed/bits/statx.h
```

Comment:
Does it need the conditional, since you're doing `-f` already?

Reply:
Yeah, should be enough. (As is not adding a -r to rm invocation)